code/game/objects/items/kitchen.dm

```diff
@@ -28,33 +28,50 @@
 	attack_verb = list("attacked", "stabbed", "poked")
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 30, "stamina" = 0)
+	var/datum/reagent/forkload //used to eat any food (who coded this to just work for omlettes?)
+
+/obj/item/kitchen/fork/suicide_act(mob/living/carbon/user)
+	user.visible_message("<span class='suicide'>[user] stabs \the [src] into [user.p_their()] chest! It looks like [user.p_theyre()] trying to take a bite out of [user.p_them()]self!</span>")
+	playsound(src, 'sound/items/eatfood.ogg', 50, 1)
+	return BRUTELOSS
+
+
+
+/obj/item/kitchen/knife/poison/attack(mob/living/M, mob/user)
+	if (!istype(M))
+		return
+	. = ..()
+	if (!reagents.total_volume || !M.reagents)
+		return
+	var/amount_inject = amount_per_transfer_from_this
+	if(!M.can_inject(user, 1))
+		amount_inject = 1
+	var/amount = min(amount_inject/reagents.total_volume,1)
+	reagents.reaction(M,INJECT,amount)
+	reagents.trans_to(M,amount_inject)
+
+/obj/item/kitchen/spoon
+	name = "spoon"
+	desc = "Not pointy."
+	icon_state = "spoon"
+	force = 5
+	w_class = WEIGHT_CLASS_TINY
+	throwforce = 0
+	throw_speed = 3
+	throw_range = 5
+	materials = list(/datum/material/iron=80)
+	flags_1 = CONDUCT_1
+	attack_verb = list("attacked", "scooped",)
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 30, "stamina" = 0)
 	var/datum/reagent/forkload //used to eat omelette
 
 /obj/item/kitchen/fork/suicide_act(mob/living/carbon/user)
```

Comment:
this one

Reply:
Probably forgot to change one of these to be the spoon's suicide_act() if I had to guess